modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -46,10 +48,11 @@ spec:
 <3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
 <4> The actual name of the Container Network Interface (CNI) plug-in that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
 <5> The name of the Linux bridge configured on the node.
-<6> Optional: The VLAN tag.
-<7> The CNI plug-in that allows the MAC pool manager to assign a unique MAC address to the connection.
+<6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
+<7> Optional: The VLAN tag.
+<8> The CNI plug-in that allows the MAC pool manager to assign a unique MAC address to the connection.
```

Comment:
This is not directly related to this change, so it may be appropriate to change it in a following PR: We no longer require `cnv-tuning` to set the MAC address, cnv-bridge now supports it natively.

Reply:
Maybe you can open a documentation bug to address that change?